clusterloader2/api/validation_test.go

```diff
@@ -264,6 +323,7 @@ func TestVerifyStep(t *testing.T) {
 				Measurements: []*Measurement{
 					{
 						Method: "test1",
+						Identifier: "measurement1",
```

Comment:
Nit: we could also add it to "has phases and measurements", to ensure that this test case fails because it has both phases and measurements, not because it has invalid measurement.

Reply:
Yea, true. The way these tests work right now is quite error-prone - we are only checking that specific configuration is correct/not correct and we don't check why.

Reply:
Done.

Reply:
Agree